packages/cli/src/constructs/frequency.ts

```diff
@@ -1,24 +1,81 @@
+/**
+ * Represents a check execution frequency with optional offset.
+ * Use the static constants for common scheduling patterns.
+ * 
+ * **Supported Frequencies:**
+ * - **API/TCP Checks**: 0 (seconds), 1, 2, 5, 10, 15, 30, 60, 120, 180, 360, 720, 1440 minutes
+ * - **Browser/Multi-step/Playwright**: 1, 2, 5, 10, 15, 30, 60, 120, 180, 360, 720, 1440 minutes
+ * - **Second Schedules** (frequency=0): 10, 20, 30 seconds only
+ * - **Default**: 10 minutes for all check types
+ * 
+ * **Frequency Offset Validation:**
+ * - For frequency=0: Must be exactly 10, 20, or 30 seconds
+ * - For frequency ≤ 60 minutes: Max offset = floor(frequency × 10) seconds  
+ * - For frequency > 60 minutes: Max offset = ceil(frequency ÷ 60) seconds
+ * 
+ * @example
+ * ```typescript
+ * // High-frequency monitoring (every 30 seconds) - API/TCP only
+ * frequency: Frequency.EVERY_30S
+ * 
+ * // Standard API monitoring (every 5 minutes)
+ * frequency: Frequency.EVERY_5M
+ * 
+ * // Daily health checks (every 24 hours)
+ * frequency: Frequency.EVERY_24H
+ * 
+ * // Custom frequency (every 45 minutes) - must be supported value
+ * frequency: 45  // Not supported - use closest: 30 or 60
+ * ```
```

Comment:
This seems "a bit much"... Should we shorten it?

Reply:
I think we could remove the `**Supported Frequencies:**` and `**Frequency Offset Validation:**` sections, and then the last `// Custom frequency` example which is kinda useless. Also, in the future we could add CLI-side validation for the number value, making the example even less useful.